lib/ansible/module_utils/basic.py

```diff
@@ -572,57 +582,45 @@ def load_file_common_arguments(self, params, path=None):
 
     def selinux_mls_enabled(self):
         if self._selinux_mls_enabled is None:
-            self._selinux_mls_enabled = HAVE_SELINUX and selinux.is_selinux_mls_enabled() == 1
-
+            self._selinux_mls_enabled = is_selinux_mls_enabled()
         return self._selinux_mls_enabled
 
     def selinux_enabled(self):
         if self._selinux_enabled is None:
-            self._selinux_enabled = HAVE_SELINUX and selinux.is_selinux_enabled() == 1
-
+            self._selinux_enabled = is_selinux_enabled()
         return self._selinux_enabled
 
     # Determine whether we need a placeholder for selevel/mls
     def selinux_initial_context(self):
         if self._selinux_initial_context is None:
-            self._selinux_initial_context = [None, None, None]
+            self._selinux_initial_context = DEFAULT_SELINUX_CONTEXT
```

Comment:
Looks like either
```suggestion
            self._selinux_initial_context = list(DEFAULT_SELINUX_CONTEXT)
```
is needed or the tests should start expecting tuples.

Reply:
im 50/50 on forcing the type as a list or tuple, since it seems to be 3 or 4 items depending on function (using/ignoring security level)